package-lock.json

```diff
@@ -926,7 +926,8 @@
       "dependencies": {
         "hoist-non-react-statics": {
           "version": "3.3.0",
-          "bundled": true,
+          "resolved": "https://registry.npmjs.org/hoist-non-react-statics/-/hoist-non-react-statics-3.3.0.tgz",
```

Comment:
are these intended changes for this PR?

Reply:
This is because `hoist-non-react-statics` is a dependency of rum-react and is generated by npm install.